src/net.h

```diff
@@ -130,6 +130,7 @@ class CConnman
 
     struct Options
     {
+        bool m_relay_txes = false;
```

Comment:
Mostly because of this: https://github.com/bitcoin/bitcoin/pull/10596#discussion_r128319151

Personally, I also slightly prefer to put the correct default here, but it does not matter much as it gets overwritten anyway.